cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -470,7 +214,111 @@ def close(self):
+
+        if file_format in ['qcow2', 'vmdk']:
+            conf_path = None
+            try:
+                with open('/etc/ceph/ceph.conf', 'r') as fd:
+                    conf = fd.read()
+                    conf = '%s\n%s\n' % (conf, 'rbd default format = 2')
+                    conf_path = linux.write_to_temp_file(conf)
+
+                shell.check_run('%s -f %s -O rbd rbd:%s rbd:%s:conf=%s' % (qemu_img.subcmd('convert'), file_format,
+                                                                           self.tmpPath, self.dstPath, conf_path))
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)